app/models/recipe.rb

```diff
@@ -37,6 +42,7 @@ class Recipe < ApplicationRecord
   # == Validations ==========================================================
 
   validates :name, presence: true
+  validates :user_id, presence: true
```

Comment:
Do not validates `user_id` column, validate relation.
```suggestion
  validates :user, presence: true
```

Reply:
changed